mirrors.yaml

```diff
@@ -20,664 +18,675 @@
-  - Zhongshan, Guangdong, China
+  - Xi'an, Shaanxi, China
+  - Xining, Qinghai, China
+  - Zhangjiakou, Hebei, China
+  - Zhengzhou, Henan, China
+  frequency: 8h
   protocols:
   - ipv4
   - ipv6
-  - http
-  - https
   geocoords:
-  - 52.3727598, 4.8936041
-  - 39.906217, 116.3912757
+  - 52.3730796, 4.8924534
+  - 13.8245796, 100.6224463
+  - 3.9767059, -73.1493675
+  - 38.3042796, 116.8511597
+  - 28.1450774, 113.2384362
   - 31.8122623, 119.9691539
+  - 30.6598628, 104.0633717
   - 41.8755616, -87.6244212
+  - 29.5656729, 106.5479189
+  - 39.740278, 122.255833
   - 32.7762719, -96.7968559
-  - 23.0205969, 113.7457788
-  - 23.1102636, 112.99801241836545
+  - 28.6273928, 77.1716954
+  - 39.7392364, -104.984862
+  - 23.0239788, 113.1159558
   - 50.1106444, 8.6820917
-  - 23.1301964, 113.2592945
+  - 30.2489634, 120.2052342
+  - 45.7593633, 126.6276177
+  - 31.8228094, 117.2218033
+  - 47.328863, 130.2816917
+  - 49.249879, 127.11731
+  - 37.7364305, 115.6677229
+  - 10.7763897, 106.7011391
   - 40.8337963, 111.6730788
   - 22.2793278, 114.1628131
-  - 23.1146515, 114.4107202
+  - 41.006381, 28.9758715
+  - -6.175247, 106.8270488
+  - 47.02157, 132.044137
+  - 36.6507007, 117.1140042
+  - 37.6866796, 112.7383414
+  - -26.205, 28.049722
+  - 3.1516964, 101.6942371
+  - 25.0363006, 102.7086139
   - 50.4500336, 30.5241361
-  - 39.5128345, 116.6997446
-  - 51.5073219, -0.1276474
+  - 6.4550575, 3.3941795
+  - 36.474436, 103.733224
+  - 24.3284746, 109.4239676
+  - 51.4893335, -0.14405508452768728
   - 40.4167047, -3.7035825
-  - -37.8142176, 144.9631608
-  - 45.5074301, 9.2446893
-  - 55.7504461, 37.6174943
-  - 19.0759899, 72.8773928
-  - 32.0437711, 118.7788734
-  - 48.8206809, 2.3635646
+  - 14.5906346, 120.9799964
+  - -37.8142454, 144.9631732
+  - 25.7741728, -80.19362
+  - 45.4641943, 9.1896346
+  - 55.625578, 37.6063916
+  - 18.9733536, 72.82810491917377
+  - 28.687547, 115.8540042
+  - 31.9827896, 120.8904588
+  - 25.061706, 121.4589414
+  - 34.6198813, 135.490357
+  - 48.8534951, 2.3483915
   - 36.0638034, 120.3781372
-  - 37.3361905, -121.890583
-  - 47.6038321, -122.3300624
-  - 31.2322758, 121.4692071
-  - 22.555454, 114.0543297
+  - -22.9110137, -43.2093727
+  - 37.3361663, -121.890591
+  - -23.5506507, -46.6333824
+  - 47.6038321, -122.330062
+  - 37.5666791, 126.9782914
+  - 31.2312707, 121.4700152
+  - 41.8026095, 123.4279105
+  - 22.5445741, 114.0545429
   - 38.0429742, 114.5088385
   - 1.357107, 103.8194992
   - 39.0017455, -77.4033349
   - 59.3251172, 18.0710935
-  - 31.3016935, 120.5810725
-  - -33.8548157, 151.2164539
-  - 39.0856735, 117.1951073
-  - 35.6828387, 139.7594549
+  - -33.8698439, 151.2082848
+  - 25.0375198, 121.5636796
+  - 37.870009, 112.5452739
+  - 39.6295951, 118.1738712
+  - 39.1175488, 117.1913008
+  - 35.6840574, 139.7744912
   - 30.5951051, 114.2999353
-  - 31.4933074, 120.3074357
-  - 33.3495657, 120.1576778
-  - 32.3969935, 119.4077008
-  - 22.5213807, 113.3656141
-- provider: 华为云
```

Comment:
The Huawei Cloud mirror has not synchronized with archlinuxcn for more than 59 days.